client/src/pages/database/index.tsx

```diff
@@ -183,39 +184,43 @@ export const Database = () => {
             Settings
           </TabNavLink>
         </TabNav>
-      </div>
+      </Container>
 
-      <div className="max-w-5xl mx-auto px-4 sm:px-6 lg:px-8">
+      <Container maxW="5xl" mt={10}>
         <div className="grid sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-2 xl:grid-cols-2 gap-4 mt-10">
           <div>
-            <h1 className="text-lg font-bold py-5">Database info</h1>
+            <Heading as="h2" size="md" py={5}>
+              Database info
+            </Heading>
             <div className="bg-gray-100 shadow overflow-hidden rounded-lg border-b border-gray-200">
-              <table className="mt-4 mb-4 min-w-full bg-white">
-                <tbody className="text-gray-700">
-                  <tr className="bg-gray-100">
-                    <td className="w-1/3 text-left py-3 px-4 font-semibold">
+              <Table mt="4" mb="4" variant="simple">
+                <Tbody mt="10">
+                  <Tr py="4">
```

Comment:
@pradel is it better to do `py={4}` or `py="4"`? 

Both seem to work, but is there any difference/preference?

Reply:
I have no idea tbh, maybe it's stated in the chakra docs?

Reply:
Merging this one, will have a look later.